src/main/scala/nl.knaw.dans.easy.properties/app/graphql/types/MutationType.scala

```diff
@@ -365,6 +374,13 @@ trait MutationType {
     resolve = ctx => DepositResolver.depositById(ctx.value.depositId)(ctx.ctx),
   )
 
+  implicit val DepositIdType: ScalarType[DepositId] = UUIDType // TODO caused compiler errors in DepositType when placed in ScalarTypes
```

Comment:
Let's look at this after you get back from holiday. Not sure if we need this line.